pkgs/by-name/zs/zs/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, fetchFromGitea, buildGoModule, installShellFiles }:
+
+buildGoModule rec {
+  pname = "zs";
+  version = "0.4.1";
+
+  src = fetchFromGitea {
+    domain = "git.mills.io";
+    owner = "prologic";
+    repo = "zs";
+    rev = version;
+    hash = "sha256-V8+p19kvVh64yCreNVp4RVdkJkjrq8Q5VbjaJWekZHY=";
+  };
+
+  vendorHash = "sha256-KXcYTYO4wnWOup5uJ6T+XwthX5l2FL02JyOt1Nv51Sg=";
+
+  ldflags = [
+    "-w"
+    "-X=main.Version=${version}"
+    "-X=main.Commit=${src.rev}"
+    "-X=main.Build=1970-01-01T00:00:00+00:00"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  doCheck = true;
+
+  checkPhase = ''
+    go test ./...
+  '';
+
```

Comment:
```suggestion
```
Not needed since `buildGoModule` runs tests by default
https://github.com/NixOS/nixpkgs/blob/e1d09cc0fd2d759880a040ec1afc55f6e5cd7121/pkgs/build-support/go/module.nix#L265-L276

Reply:
removed it 👍